modules/ROOT/pages/faq.adoc

```diff
@@ -193,3 +193,20 @@ Kubernetes cluster, you will be able to use upcoming tools to coordinate
 updates and reboots.
 
 https://discussion.fedoraproject.org/t/launch-faq-how-do-i-coordinate-cluster-wide-os-updates-is-locksmith-or-the-container-linux-update-operator-available-for-fedora-coreos/56[discuss]
+
+== How do I upload Fedora CoreOS to private AWS EC2 regions?
+
+Fedora CoreOS today is only uploaded to the public AWS regions. For private
+regions like GovCloud and AWS China, you must upload the images yourself.
```

Comment:
s/public/standard/
`For regions in other AWS partitions like GovCloud and AWS China`...